core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -43,6 +45,55 @@ THE SOFTWARE.
         <l:task href="${rootURL}/${it.viewUrl}newJob" icon="icon-new-package icon-md" it="${app}" permission="${it.itemCreatePermission}" title="${%NewJob(it.newPronoun)}"/>
       </st:include>
 
+      <l:task href="#modal" icon="icon-new-package icon-md" title="New Item Popup"/>
+      <!-- <st:include page="popup.jelly" optional="true" /> -->
+
+      <st:include optional="true">
```

Comment:
Apparently, nothing. I though I needed to include html between these tags. However, it seems that I dont. I will remove them.